docs/source/en/using-diffusers/pipeline_techniques.md

```diff
@@ -0,0 +1,186 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Enhance image quality
```

Comment:
I'm not totally in love with this title so I'm open to any suggestions you may have!

Reply:
"Controlling image quality"?

Reply:
Yeah that could be good as well, thanks! :) I'll leave this PR open a few more days to allow others to chime in before merging.